apps/docs/src/app/core/component-docs/grid-list/examples/default/grid-list-example.component.html

```diff
@@ -1,25 +1,36 @@
 <fd-grid-list>
-    <fd-grid-list-title-bar title="Products">
+    <fd-grid-list-title-bar [title]="'Products'">
```

Comment:
do you know why the binding is not working? It should work with `title="Products"`

Reply:
The simple binding works but shows a hover tooltip across the entire title bar, which I was trying to avoid.